src/components/body/body.component.ts

```diff
@@ -335,7 +335,7 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
       offset = Math.ceil(offset);
     }
 
-    if (direction !== undefined && !isNaN(offset)) {
+    if (this.scrollbarV && direction !== undefined && !isNaN(offset)) {
       this.page.emit({ offset });
```

Comment:
@vixriihi This is a page event, not scroll event.

Reply:
true and it's triggered from onBodyScroll.

Reply:
I did notice that the scroller component does not turn off scroll events when the `scrollbarV` or `scrollbarH` options changes

```
  ngOnInit(): void {
    // manual bind so we don't always listen
    if (this.scrollbarV || this.scrollbarH) {
      this.parentElement = this.element.parentElement.parentElement;
      this.onScrollListener = this.renderer.listen(
        this.parentElement, 'scroll', this.onScrolled.bind(this));
    }
  }
```

This can be moved and check based on a virtual property, and listen and stop listening based on the property